tools/apiview/emitters/cadl-apiview/package.json

```diff
@@ -53,11 +53,8 @@
     "dist/**",
     "!dist/test/**"
   ],
-  "dependencies": {
-    "@azure-tools/cadl-azure-core": "~0.25.0",
-    "@azure-tools/cadl-dpg": "~0.25.0",
+  "peerDependencies": {
```

Comment:
How does peerDependencies actually help with our problems?

Reply:
We should try to avoid using `peerDependencies` unless absolutely necessary, since many package managers do not handle them well.  So I agree with Wes, I'd like to know more about why we think we need to use `peerDependencies` and what problem we are trying to solve.